### DISCUSSION TITLE
umi 正式环境下 如何使用  proxy呢

### DISCUSSION THREAD

**Discussion by soxft** [umijs/umi#12706] [Q&A] [answered]
UMI 可以在 .umirc.ts 中 配置 proxy 字段，来实现代理

```json
{
  "scripts": {
    "dev": "umi dev",
    "build": "umi build",
    "postinstall": "umi setup",
    "setup": "umi setup",
    "start": "npm run dev"
  }
}
```
 但是似乎在默认的 package.json 中, 没有找到 类似 NextJs 类似的 正式运行(next start) 命令，只有在 `dev` 下才能使用 proxy

正式环境下 似乎只能通过打包成 dist 后，通过 静态资源的形式 访问

**Comment by soxft** [comment]
目前是 直接部署到 vercel的，也可以通过 vercel.json 或者 serverless function 来实现类似的 proxy

**Comment by xiaohuoni** [comment]
https://umijs.org/docs/api/commands#preview

**Reply by soxft** [reply, marked as answer]
> https://umijs.org/docs/api/commands#preview

感谢
